monkestation/code/modules/mob/living/carbon/human/species_type/oni.dm

```diff
@@ -0,0 +1,106 @@
+/datum/species/oni
+	name = "\improper Oni"
+	plural_form = "Onis"
+	id = SPECIES_ONI
+	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC | MIRROR_PRIDE | ERT_SPAWN | RACE_SWAP
+	sexes = TRUE
+	species_traits = list(
+		MUTCOLORS,
+		LIPS,
+		EYECOLOR,
+		HAIR,
+		FACEHAIR
+	)
+	inherent_biotypes = MOB_ORGANIC | MOB_HUMANOID
+	inherent_traits = list(
+		TRAIT_HARDLY_WOUNDED,
+	)
+	external_organs = list(
+		/obj/item/organ/external/oni_tail = "normal",
+		/obj/item/organ/external/oni_horns = "normal",
+		/obj/item/organ/external/goblin_ears = "long",
+		)
+	mutantlungs = /obj/item/organ/internal/lungs/oni
+	disliked_food = VEGETABLES | GROSS
+	liked_food = GORE | MEAT | SEAFOOD
+	maxhealthmod = 1.1
+	stunmod = 1.2
+	speedmod = +0.1
```

Comment:
```suggestion
	speedmod = 0.1
```

Reply:
All I can see that's a code issue. It's probably a good idea to develop the species a bit more in this stage because they're not really distinct. Changing species balance after they're in is an absolute nightmare because people get attached to the current version.